src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
Maybe you should use profiles, or H2 is your production database?

Reply:
I used H2 on this phase just to see if relations are created as I imagined after app build. Later on i will change it for MySQL or create 2 different profiles for dev and production